src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/BulkPrintController.java

```diff
@@ -65,36 +67,53 @@
 
         validateCaseData(callback);
 
-        CaseDetails caseDetails = callback.getCaseDetails();
+        caseDetails = callback.getCaseDetails();
+
+        generateCoversheetForApplicant(authToken);
```

Comment:
In my opinion both of these should return a modified copy of the case data. This is not a show stopper, though.